src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -837,6 +885,7 @@ private void ApplyRename(Solution newSolution, IUIThreadOperationContext operati
                 finalSolution = finalSolution.WithDocumentName(id, newDocument.Name);
```

Comment:
I can't add a comment, but above there are a bunch of GetXSynchronously(CancellationToken.None).  Think they can be made async + take in the cancellation token?

Reply:
good call.  will do.

Reply:
Done!